src/Main.java

```diff
@@ -1,5 +1,73 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, fdgdfWorld!");
+        System.out.println("Задача 1");
+        int clientOS = 1;
+        if (clientOS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else if (clientOS == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        } else {
+            System.out.println("значение отсутствует");
+        }
+        System.out.println("Задача 2");
+        clientOS = 1;
+        int clientDeviceYear = 2018;
+        if (clientOS == 0 && clientDeviceYear < 2015) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        } else if (clientOS == 0 && clientDeviceYear >= 2015) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else if (clientOS == 1 && clientDeviceYear < 2015) {
+            System.out.println("Установите облегченную версию приложения для Android по ссылке");
+        } else {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+        System.out.println("Задача 3");
+        int year = 2015;
+        if ((year >= 1584 && year % 4 == 0) || (year % 100 != 0 && year % 400 == 0)) {
```

Comment:
неправильно расставлены скобки, а также условие неверное
```suggestion
        if (year >= 1584 && year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
```